src/remediations/remediations.queries.js

```diff
@@ -72,7 +76,62 @@ function resolvedCountSubquery () {
     );
 }
 
-exports.list = function (
+function findRunStatus(run) {
+    if (run.count_failure > 0 || run.count_timeout > 0) {
+        return FAILURE;
+    } else if (run.count_running > 0 && run.count_timeout === 0 && run.count_failure === 0) {
+        return RUNNING;
+    } else if (run.count_success > 0 && run.count_timeout === 0) {
+        return SUCCESS;
+    }
+}
```

Comment:
This code exists somewhere else, too.  We should just have it in one place instead of duplicating it here (although I prefer your suggestion to hold off on sorting by status until we can extend the db schema to start tracking that locally.)